### PR TITLE
Create open-science-for-bioinformaticians blogpost

### PR DESCRIPTION
**Pull request by seunolufemi123**
:+1::tada: First of all, thanks for taking the time to contribute! :tada::+1:

## FOR CONTRIBUTOR

* [ ] I have read the [CONTRIBUTING.md](https://github.com/open-life-science/open-life-science.github.io/blob/main/CONTRIBUTING.md) document

PR Type: 
* [x] This PR adds a new blog post
* [ ] This PR does something else (explain above)

 
## FOR REVIEWERS

Thanks for taking the time to review! :heart:

Here are the list of things to make sure of:
* [x] The website builds (a check will fail if not)
* [ ] All images have been added within the Pull Request and they have Alt text
* [ ] If there are paragraphs or text, the key messages are highlighted
* [ ] All internal links (within OLS website) use the [`{% link path_to_file.md %}` format](https://jekyllrb.com/docs/liquid/tags/#link)
* [ ] The preview corresponds to the changes described in the Pull Request
* [ ] The code is tidy and passes the linting tests

### CODE REVIEW

File: _posts/open-science-for-bioinformaticians [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
```
Comment (yochannah) [inline _posts/open-science-for-bioinformaticians:1]: Filename - two things needed: 
- add `.md` to the end - this is needed so the website knows to turn it into a post. 
- add a date to the start, e.g. `2024-20-21-open-science[etc...].md`

Comment (NPDebs) [reply _posts/open-science-for-bioinformaticians:1]: Thanks, Yo. 🙏 

@seunolufemi123 - The naming convention is `[year]-[month]-[day]-[post-name].md.`

File: _posts/2024-10-21-open-science-for-bioinformaticians.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+Over the two days, participants engaged in sessions that covered:
+- **Open Science Fundamentals**: Understanding the benefits and practical applications.
+- **Data Management in Bioinformatics**: Tools and platforms for effective open data sharing.
+- **Ethical and Legal Considerations**: Navigating data ownership and responsible sharing.
+- **Collaborative Tools**: Practical demonstrations of open science platforms for teamwork.
+
+![Alt Text](/images/2022-04-01-header-image-speed-blod-nadine.jpg)
+
+The program included interactive activities that allowed participants to apply the concepts discussed during the sessions. Each day concluded with a lively Q&A, where attendees engaged facilitators on topics ranging from licensing issues to choosing appropriate open-access repositories. The sessions were moderated by Seun Olufemi and Emmanuel Adamolekun, and featured expert practitioners as speakers and facilitators from the global open science community, including Dr. Malvika Sharan, Laura Ondari, Pauline Karega, Michael Landi, Dr. Sara El-Gebali, Dr. Yo Yehudi, and Deborah Udoh, who provided valuable insights and hands-on guidance.
```
Comment (NPDebs) [inline _posts/2024-10-21-open-science-for-bioinformaticians.md:31]:
```suggestion
The program included interactive activities that allowed participants to apply the concepts discussed during the sessions. Each day concluded with a lively Q&A, where attendees engaged facilitators on topics ranging from licensing issues to choosing appropriate open-access repositories. The sessions were moderated by Seun Olufemi and Emmanuel Adamolekun, and featured expert practitioners as speakers and facilitators from the global open science community, including [Dr. Malvika Sharan]({% link people.md %}#malvikasharan), [Laura Ondari]({% link people.md %}#laurah-nyasita-ondari), [Pauline Karega]({% link people.md %}#karegapauline), [Michael Landi]({% link people.md %}#landimi2), [Dr. Sara El-Gebali]({% link people.md %}#selgebali), [Dr. Yo Yehudi]({% link people.md %}#yochannah), and [Deborah Udoh]({% link people.md %}#npdebs), who provided valuable insights and hands-on guidance.
```

Comment (NPDebs) [reply _posts/2024-10-21-open-science-for-bioinformaticians.md:31]: This makes it easy to link to the speakers' profiles (on our /people page).

**Review by NPDebs — COMMENTED** [review]
Hi @seunolufemi123. 👋🏽 
Thanks for not giving up on your PR. :-) It wasn't working right because the file name didn't follow the naming convention. That is, `year-month-day-title-of-post.md`.

I'll just re-echo Yo's comment about the Alt text for images. Let me know if you need some guidance on that.

Once we sort that out, I'm happy to merge.

**Review by NPDebs — APPROVED** [review]
Hi @seunolufemi123, are you happy with how I've implemented the changes you requested?

With your blessing, I'm happy to merge.

**Pull request merged**